src/containers/Openings/index.tsx

```diff
@@ -26,8 +26,12 @@ const Openings: FC = () => {
   const [categoryFilter, setCategoryFilter] = useState<OpeningCategory>(OpeningCategory.all);
 
   useEffect(() => {
-    setCategoryFilter(categoryParam);
-  }, [categoryParam]);
+    if (OPENING_CATEGORY_FILTERS.some((filter) => filter.pathname === categoryParam)) {
+      setCategoryFilter(categoryParam);
+    } else {
+      history.push('/openings/All');
```

Comment:
lets use history.redirect to keep the history clean

Reply:
done